crates/uv/tests/pip_compile.rs

```diff
@@ -1598,8 +1594,8 @@ dependencies = ["django==5.0b1", "django==5.0a1"]
 
     ----- stderr -----
       × No solution found when resolving dependencies:
-      ╰─▶ my-project cannot be used because there are conflicting versions for
-          `django`: `django==5.0b1` does not intersect with `django==5.0a1`
+      ╰─▶ Because my-project depends on django==5.0b1 and my-project depends on
+          django==5.0a1, we can conclude that the requirements are unsatisfiable.
```

Comment:
Nice. It'll be better once we combine clauses #1009, maybe someday we should even track the source of dependencies so we can say what file each of these came from.